packages/core-orders/src/module/configureOrdersModule-processing.ts

```diff
@@ -248,6 +248,43 @@ export const configureOrderModuleProcessing = ({
       return updatedOrder;
     },
 
+    reject: async (order, params, requestContext) => {
+      const { modules } = requestContext;
+      const orderId = order._id;
+
+      if (order.status !== OrderStatus.PENDING) return order;
+
+      let updatedOrder = await modules.orders.updateContext(
+        orderId,
+        params.orderContext,
+        requestContext,
+      );
+      updatedOrder = await modules.orders.updateStatus(
+        orderId,
+        { status: OrderStatus.REJECTED, info: 'rejected manually' },
+        requestContext,
+      );
+      updatedOrder = await modules.orders.processOrder(
+        updatedOrder,
+        {
+          paymentContext: params.paymentContext,
+          deliveryContext: params.deliveryContext,
+        },
+        requestContext,
+      );
+      const orderPayment = await modules.orders.payments.findOrderPayment({
+        orderPaymentId: order.paymentId,
+      });
+      await modules.payment.paymentProviders.cancel(
```

Comment:
Does this also refund through Postfinance when the payment has already been paid?

Reply:
Now it does :)